cli/cmd/vuln_host.go

```diff
@@ -753,14 +837,14 @@ func buildVulnHostsDetailsTable(filteredCves []api.HostVulnCVE) string {
 			}
 		} else {
 			rows := hostVulnCVEsTableForHostView(filteredCves)
-
+			array.Sort2DDescending(rows)
```

Comment:
This might break the previous sorting by severity inside `hostVulnCVEsTableForHostView()`